docs/preset-env.md

```diff
@@ -302,7 +333,18 @@ var b = new Map();
 
 #### `useBuiltIns: false`
 
-Don't add polyfills automatically per file, or transform `import "@babel/polyfill"` to individual polyfills.
+Don't add polyfills automatically per file, and don't transform `import "core-js"` or `import "@babel/polufill"` to individual polyfills.
+
+### `corejs`
+
+`2`, `3` or `{ version: 2 | 3, proposals: boolean }`, defaults to `2`.
+
+This option only has an effect when used alongside `useBuiltIns: usage` or `useBuiltIns: entry`, and ensures `@babel/preset-env` injects the correct imports for your `core-js` version.
+
+By default, only polyfills for stable ECMAScript features are injected: if you want to polyfill them, you have three different options:
+- set the [`shippedProposals`](#shippedproposals) option to `true`. This will enable polyfills and transforms for proposal which have already been shipped in browsers for a while.
+- use `corejs: { version: 3, proposals: true }`. This will enable polyfilling of every proposal supported by `core-js`.
+- when using `useBuiltIns: "entry"`, you can directly import a [proposal polyfill](https://github.com/zloirock/core-js/tree/master/packages/core-js/proposals): `import "core-js/proposals/string-replace-all"`.
```

Comment:
I am not sure that this is a clear enough wording. For polyfilling proposals, `shippedProposals` and `proposals` flags make sense only for `usage`. For `entry`, makes sense the third option (and not only from `proposals` path - for example, `import "core-js"` includes import of all polyfills) without any flags.